data/intro missions.txt

```diff
@@ -451,3 +451,139 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro: Follow Up - Hai Before Main Plot"
+	name "Bring James to <planet>"
+	description "Take James to <destination> to meet the Hai."
+	passengers 1
+	landing
+	source "Hestia"
+	destination "Allhome"
+	to offer
+		or
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Interceptor]: done"
+		has "First Contact: Hai: offered"
+		not "chosen sides"
+	on offer
+		conversation
+			`As you're coming in for a landing, you remember that James left you an open invitation to look him up. Do you want to check in and see how he is doing?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	When you contact James to tell him that you are on visiting <origin>, he is happy to come out to the starport and meet you. The two of you talk a bit about the business of running ships and before long the conversation comes around to the Hai. James is not completely surprised as your story confirms rumors that he has already heard.`
+			choice
+				`	"Why don't you come with me and see them for yourself? There are a lot of retired spacers on <planet>."`
+				`	"It has been good catching up with you, James."`
+					goto end
+			`	James agrees. When you ask if he needs to collect any of his belongings before the trip he declines, but buys a simple traveling hygiene kit from one of the spaceport shops.`
+				accept
+			label end
+			`	You finish your conversation and go on your way`
+				decline
+	on decline
+		log `Paid a visit to James on <origin> and told him about the Hai.`
```

Comment:
This mission choice permanently closes options for the future.  

Probably more missions deserve log entries -- certainly not all options -- but anything that is part of a chain that needs to be followed up on, or closes off future options should be recorded.

It is easy enough to remove these, but since they do close off future options and provide a reference of where you left the NPC, it seems worth having.

Reply:
>This mission choice permanently closes options for the future.
>Probably more missions deserve log entries...

Then that is a matter of opening a new PR to do this, but as of right now this doesn't follow the current style of when logs are given.